packages/opentelemetry/lib/tracing/serverInstrumenation.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { ExpressInstrumentation } from '@opentelemetry/instrumentation-express';
+import { InstrumentationBase } from '@opentelemetry/instrumentation';
+import _ from 'lodash';
+
+class ServerInstrumentation {
+  constructor () {
+    this._httpInstrumentation = new HttpInstrumentation();
+    this._expressInstrumentation = new ExpressInstrumentation();
+    this._httpCurrentConfig = {
+      ignoreIncomingPaths: [],
+      ignoreOutgoingUrls: [],
+      applyCustomAttributesOnSpan: {}
+    };
+  }
+
+  updateCurrentConfig () {
+    this._httpInstrumentation.setConfig(this._httpCurrentConfig);
+  }
+
+  /**
+   * get the server instrumentation instances as an array
+   * @return {[InstrumentationBase]}
```

Comment:
extra square brackets

Reply:
this returns a list of InstrumentationBase

Reply:
then the syntax would be `{InstrumentationBase[]}` See https://jsdoc.app/tags-param.html for more details